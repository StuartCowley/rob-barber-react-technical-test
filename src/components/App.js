import React, { useState } from "react";
import "../styles/app.css";
import Search from "./Search";
import SearchResults from "./SearchResults";

function App() {
  const [searchResults, setSearchResults] = useState([]);

  return (
    <div className="app">
      <img
        className="nasa-logo"
        src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a3/NASA_Worm_logo.svg/1280px-NASA_Worm_logo.svg.png"
        alt="nasaLogo"
      />
      <Search setSearchResults={setSearchResults} />
      <SearchResults results={searchResults} />
    </div>
  );
}

export default App;