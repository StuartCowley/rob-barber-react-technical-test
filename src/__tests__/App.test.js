import { render, screen } from '@testing-library/react';
import App from '../components/App';

describe("app", () => {
const { asFragment } = render(<App />)

it("renders correctly", () => {
   expect(asFragment()).toMatchSnapshot();
});

});