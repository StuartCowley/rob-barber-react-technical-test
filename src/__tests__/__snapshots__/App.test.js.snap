// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app renders correctly 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <img
      class="nasa-logo"
      src="https://cdn.cnn.com/cnnnext/dam/assets/200424060716-nasa-worm-logo.jpg"
    />
    <input
      class="search-field"
      type="text"
    />
  </div>
</DocumentFragment>
`;
